modules/local/prefetchchecker/main.nf

```diff
@@ -11,11 +11,19 @@ process PREFETCH_CHECKER {
     exec:
     task.workDir.resolve("failures_report.csv").withWriter { writer ->
 
-        writer.writeLine("sample,error_accession")  // header
+        sample_name = false
+        failures.each { if ( it[0].id != null) {
```

Comment:
Could you adjust the indentation/whitespace here so that it's a bit easier to read. Specifically, put the `if`-statement on it's own newline like:

```groovy
failures.each {
    if ( it[0].id != null) {
        sample_name = true
    }
}
```

Reply:
261f455